server/command.go

```diff
@@ -264,10 +278,20 @@ func (p *Plugin) executeUnlinkCommand(args *model.CommandArgs) (*model.CommandRe
 		return &model.CommandResponse{}, nil
 	}
 
+	if _, err = p.API.UnshareChannel(link.MattermostChannelID); err != nil {
```

Comment:
Should this be conditional on the configuration setting?

Reply:
I don't think we need to make it conditional.